tests/cpp/test_scatter_gather.cpp

```diff
@@ -1255,8 +1255,12 @@ TEST_F(ScatterGatherTest, GatherIterGoupedReduction) {
   at::Tensor input_idx = at::randint(0, input_dims[dim], index_dims, options_i);
   std::vector<c10::IValue> aten_inputs = {input, input_idx};
 
-  auto rparams = getReductionHeuristics(&fusion, aten_inputs);
-  NVF_CHECK(rparams, "Reduction schedule was not generated!");
+  auto reduction_scheduler =
```

Comment:
IIUC, here we are manually modifying the generated heuristics for testing, so we can't use `scheduleWith`

Reply:
Yes.